cmd/debugger/main.go

```diff
@@ -320,7 +347,7 @@ func main() {
 		os.Exit(exitCode)
 	}
 
-	log.With("command", args).With("version", Version).Debug("running command")
+	log.With("command", args).With("version", Version).Debug("earthly-debugger: running command")
```

Comment:
switch this to consologger to be consistent?

Reply:
yeah, I thought it will be fine since it's debug level, but I suppose we can verbose print this one